stream_alert_cli/athena/helpers.py

```diff
@@ -47,8 +86,25 @@ def to_athena_schema(log_schema):
             athena_schema[key_name] = SCHEMA_TYPE_MAPPING[list]
         elif isinstance(key_type, dict):
             # For recursion
-            athena_schema[key_name] = to_athena_schema(key_type)
+            athena_schema[key_name] = logs_schema_to_athena_schema(key_type)
         else:
             athena_schema[key_name] = SCHEMA_TYPE_MAPPING[key_type]
 
     return athena_schema
+
+
+def unique_values_from_query(query_result):
+    """Simplify Athena query results into a set of values.
+
+    Useful for listing tables, partitions, databases, enable_metrics
+
+    Args:
+        query_result (dict): The result of run_athena_query
+
+    Returns:
+        set: Unique values from the query result"""
+    return {
+        res
```

Comment:
Can we just call this `val` or `value`? In my mind, it makes it clearer where it comes from and matches the description

Reply:
yep I can make that change @austinbyers (assuming you mean `val` instead of `res`?)